src/cls/rgw/cls_rgw.cc

```diff
@@ -3775,13 +3775,14 @@ static int rgw_cls_lc_list_entries(cls_method_context_t hctx, bufferlist *in,
       /* try backward compat */
       pair<string, int> oe;
       try {
+	iter = it->second.begin();
```

Comment:
Probably doesn't matter, but the initial set above is a const iter (cbegin(), not begin())

Reply:
since it doesn't affect compilation (you can assign a non-const iterator to a const_iterator), lets just get this tested and merged